libbeat/scripts/load-dashboards.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+# Usage examples:
+# env KIBANA_INDEX='.kibana_env1' ./load.sh
+# ./load.sh -url http://test.com:9200
+# ./load.sh -url http://test.com:9200 -user admin:secret
+# ./load.sh -url http://test.com:9200 -index .kibana-test
+
+# The default value of the variable. Initialize your own variables here
+ELASTICSEARCH=http://localhost:9200
+CURL=curl
+KIBANA_INDEX=".kibana"
+KIBANA_PATH?=""
```

Comment:
Here is the PR to move the dashboards: https://github.com/elastic/beats/pull/1109 But before we have the new tooling in place with dashboards, we should keep using beats-dashboards for releases. This change is mainly to simplify the development (especially for metricbeat). It means we have to currently keep the two repos in sync, but as there are only few changes to the dashboards at the moment, it shouldn't be a problem.